### ISSUE TITLE
VS2019 Xamarin.iOS build logging verbosity creep.

### ISSUE THREAD

**Issue by plynkus** [xamarin/xamarin-macios#6651]
### Steps to Reproduce

1.  Use a recent VS2019 release (~last half year or so, certainly repros on 16.2.0).
2. Configure your IDE for minimal build/output verbosity (Tools -> Options -> Project and Solutions -> Build and Run).
3.  Build a Xamarin.iOS project when connected to a build host.
4.  Observe the build output,

### Expected Behavior

Minimal logging that's in family with other platform integrations.

### Actual Behavior

Increasing spam creep over time --- in addition to "Connecting to Mac server..." lines, there now exists an additional (and more egregious) multi-line emit from a SayHello task invocation, e.g.:

```
2>------ Rebuild All started: Project: FOO, Configuration: Debug iPhone ------
2>  Executing SayHello Task to establish a connection to a Mac Server. 
2>  						 Properties: SessionId=4ccc8445143924d735fce04eddfaaccc, 
2>  							ServerPort=50018, 
2>  							ServerAddress=xx.xx.xx.xx, 
2>  							ServerUser=USER, 
2>  							ServerPassword=, 
2>  							SSHKey=, 
2>  							SSHPassPhrase=,
2>  							AppName=FOO,
2>  							ContinueOnDisconnected=False
2>  Connecting to Mac server xx.xx.xx.xx...
```

### Environment

```
Microsoft Visual Studio Community 2019
Version 16.2.0
VisualStudio.16.Release/16.2.0+29123.88
Microsoft .NET Framework
Version 4.8.03752

Installed Version: Community

Visual C++ 2019   00435-60000-00000-AA604
Microsoft Visual C++ 2019

Application Insights Tools for Visual Studio Package   9.1.00611.1
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2019   16.2.290.48383
ASP.NET and Web Tools 2019

Azure App Service Tools v3.0.0   16.2.290.48383
Azure App Service Tools v3.0.0

C# Tools   3.2.0-beta4-19359-03+15b43b33901c88f68ef43f8314b5a2457716780d
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Extensibility Message Bus   1.2.0 (d16-2@8b56e20)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

IntelliCode Extension   1.0
IntelliCode Visual Studio Extension Detailed Info

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual C++ Wizards   1.0
Microsoft Visual C++ Wizards

Microsoft Visual Studio VC Package   1.0
Microsoft Visual Studio VC Package

Mono Debugging for Visual Studio   16.2.6 (4cfc7c3)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   5.2.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

Test Adapter for Boost.Test   1.0
Enables Visual Studio's testing tools with unit tests written for Boost.Test.  The use terms and Third Party Notices are available in the extension installation directory.

Test Adapter for Google Test   1.0
Enables Visual Studio's testing tools with unit tests written for Google Test.  The use terms and Third Party Notices are available in the extension installation directory.

TypeScript Tools   16.0.10627.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   3.2.0-beta4-19359-03+15b43b33901c88f68ef43f8314b5a2457716780d
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools 10.4 for F# 4.6   16.2.0-beta.19321.1+a24d94ecf97d0d69d4fbe6b8b10cd1f97737fff4
Microsoft Visual F# Tools 10.4 for F# 4.6

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Tools for CMake   1.0
Visual Studio Tools for CMake

Visual Studio Tools for CMake   1.0
Visual Studio Tools for CMake

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

Xamarin   16.2.0.90 (d16-2@ba267630e)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   16.2.0.325 (remotes/origin/d16-2@f10cfbf83)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   16.3.117 (59a59e8)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   9.4.0.51 (d16-2/9fa7775)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: mono/mono/2019-02@e6f5369c2d2
    Java.Interop: xamarin/java.interop/d16-2@d64ada5
    LibZipSharp: grendello/LibZipSharp/d16-2@caa0c74
    LibZip: nih-at/libzip/rel-1-5-1@b95cf3f
    ProGuard: xamarin/proguard/master@905836d
    SQLite: xamarin/sqlite/3.27.1@8212a2d
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-2@6f6c969


Xamarin.iOS and Xamarin.Mac SDK   12.14.0.110 (a8bcecc)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

### Build Logs

See example above.

> VS bug [#952688](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/952688), VS bug [#952689](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/952689)

**Comment by plynkus** [comment]
Obviously, this is a bit of nitpicking.  But given a solution with many iOS projects, build logs are now dominated by all of this information.  I think these logging entries are improperly categorized to occur under minimal logging, honestly.

**Comment by mauroa** [comment]
Thanks @plynkus, we are logging those meesages with high importance, that's why you see them with any verbosity. The reason is that the connection to a Mac is the first thing that MSBuild does prior to execute the real iOS build, which is performed remotely in the Mac. We need to keep tracking of the connection information, specially in case something goes wrong.
You're right it could become annoying if you have many iOS projects, so we'll consider this and try to improve the experience.
Finally, I encourage you to use VS Feedback (Developer Community) to submit a ticket the next time, given it's our standard way to open/handle VS issues.
Thanks

**Comment by mauroa** [comment]
We've already changed the verbosity of the logs related to connection so we don't show them unless you use a verbose build. The changes will be available on the next VS 16.6 Preview. Thanks.
